config/jupyterhub_config.py

```diff
@@ -0,0 +1,26 @@
+"""
+This is the JupyterHub configuration file. It is used to configure the JupyterHub server.
+Refer to the JupyterHub documentation for more information:
+https://jupyterhub.readthedocs.io/en/latest/tutorial/getting-started/config-basics.html
+"""
+import os
+
+from jupyterhub_config.custom_spawner import VirtualEnvSpawner
+
+c = get_config()
+
+# Set the authenticator class
+# TODO: Change the authenticator class to a secure one (e.g. GitHubOAuthenticator)
+c.JupyterHub.authenticator_class = 'jupyterhub.auth.DummyAuthenticator'
+c.Authenticator.allowed_users = {'spark_user', 'test_user1', 'test_user2'}
```

Comment:
Non-existent system users (`test_user1`, `test_user2`) will be automatically created later in `custom_spawner.py`.

Reply:
Eventually I want to switch to OAuth and phase out user/password auth completely. Maybe only allow admin users to use password.

Reply:
Presumably long term we'd swap out all the auth systems for KBase auth

Reply:
yea. That's the plan to use KBase auth eventually.